revolv/templates/project/project.html

```diff
@@ -2,8 +2,13 @@
 {% load staticfiles %}
 
 {% block head %}
+    <link rel="stylesheet" type="text/css" href="{% static "dashboard.css" %}" />
     <link rel="stylesheet" type="text/css" href="{% static "auth.css" %}">
     <link rel="stylesheet" type="text/css" href="{% static "project.css" %}">
+
+    <script type="text/javascript" src="{% static "js/vendor/d3.min.js" %}"></script>
+    <script type="text/javascript" src="{% static "js/vendor/svg4everybody.min.js" %}"></script>
```

Comment:
Don't think you actually need svg4everybody here, it's just for the svg `use=...` stuff we have/used to have on the homepage